.circleci/config.yml

```diff
@@ -28,6 +28,10 @@ jobs:
           name: Install Sourcery
           command: brew install sourcery
 
+      - run:
+          name: Run Bootstrap OSS
+          command: make bootstrap-oss
```

Comment:
let's see how circle CI does with this. if it chokes I'll split it into multiple steps that skips the module map stuff.